src/components/badge/index.js

```diff
@@ -58,6 +63,8 @@ export default class Badge extends BaseComponent {
 
   static defaultProps = {
     size: 'default',
+    pimple: false,
```

Comment:
I would remove default values that are redundant
label by default is empty
and pimple by default is undefined (which is falsy)

Reply:
A badge without a label results in an undefined call to label.length which calculates how many chars you have inside the badge. this is the only reason I have to give the label an empty string as a default prop.
I imagined that this logic is intentional and didn't want to change it

Reply:
regarding the pimple obviously you are correct :P